docs/openapi/generate-client.md

```diff
@@ -20,9 +20,44 @@ info:
 
 ### Generating your client
 
-1. Run `yarn backstage-repo-tools package schema openapi generate client --client-package <directory>`. This will create a new folder in `<directory>/src/generated` to house the generated content.
-2. You should use the generated files as follows,
+1. Run `yarn backstage-repo-tools package schema openapi generate --client-package <directory>`. This will create a new folder in `<directory>/src/schema/openapi/generated` to house the generated content. We recommend that the client package be your plugin's common package. You should then add a new entry point into the package so that the generated content can be accessed like so, `<plugin>-common/client`. To do that, adjust your `package.json` like so,
```

Comment:
Can remove all of this now right?

Reply:
Updated this doc to match the new path `src/schema/openapi/generated`, but removed all mention of the intermediate `-common/client` package